Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10470,13 +10470,17 @@ def filter_incidents_by_duplicates_and_limit(incidents_res, last_run, fetch_limi
     :return: List of incidents after filtering duplicates when len(incidents) <= limit
     :rtype: ``list``
     """
+    demisto.debug('filtering incidents by duplicates and limit')
     found_incidents = last_run.get('found_incident_ids', {})
 
     incidents = []
+    
+    demisto.debug(f'number of incidents before filtering: {len(incidents_res)}')
```

Comment:
don't use fstring in CSP (it should be compatible with python 2) - relevant for all the next changes as well